homeassistant/const.py

```diff
@@ -24,6 +24,8 @@
 CONF_PASSWORD = "password"
 CONF_API_KEY = "api_key"
 CONF_ACCESS_TOKEN = "access_token"
+CONF_COMMUNITY = "community"
```

Comment:
These two variables are very specific, as they won't be shared, better to keep them in the SNMP platform.